res/img/feather-customised/explore.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
`feather-customised` is meant to be icons from https://feathericons.com/ that we are either using directly or with small tweaks. Are these from that set? They look like custom icons to me... If custom, please move them somewhere else, such as the root `img` folder or some other subdirectory.

Reply:
I am just going off the Figma. https://www.figma.com/file/KhVjYAzy4oX3eIqwu9KQxN/Design-Kit?node-id=1065%3A84 

![image](https://user-images.githubusercontent.com/2403652/78669172-c3679700-78d3-11ea-8b7f-7d4b4ec1575a.png)

Reply:
Ah okay, I see. Well, not sure if it's actually true, but since it matches the convention design is using, I suppose we may as well leave it here then.